config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presubmits.yaml

```diff
@@ -13,3 +13,87 @@ presubmits:
         - runner.sh
         args:
         - ./scripts/ci-test.sh
+  
+  - name: pull-cluster-api-provider-vsphere-e2e
+    labels:
+      preset-dind-enabled: "true"
+      preset-service-account: "true"
+    always_run: true
+    max_concurrency: 1
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20181120-dea0825e3-master
+        args:
+        - "--job=$(JOB_NAME)"
+        - "--root=/go/src"
+        - "--repo=sigs.k8s.io/cluster-api-provider-vsphere=$(PULL_REFS)"
+        - "--repo=figo/cluster-api-provider-vsphere-ci=master"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--scenario=execute"
+        - "--"
+        - "bash"
+        - "--"
+        - "-c"
+        - "cd ./../../figo/cluster-api-provider-vsphere-ci/prow && e2e.sh ${PULL_PULL_SHA}"
+        # we need privileged mode in order to do docker in docker
+        securityContext:
+          privileged: true
+        env:
```

Comment:
you might want to consider moving some of these into a preset.

Reply:
or multiple presets.

not necessary, but may make future jobs easier to write and maintain

Reply:
@BenTheElder thank you, are you referring to those `env`? i removed `PROW_BUILD` and use `PROW_JOB_ID` instead, others are secrets, probably will not qualify for preset,  btw, i would like to learn how to add preset at Prow in general.

Reply:
@BenTheElder @cjwagner thank you, updated now.